packages/abap-deploy-config-inquirer/src/error-handler.ts

```diff
@@ -0,0 +1,25 @@
+import { PromptState } from './prompts/prompt-state';
+import LoggerHelper from './logger-helper';
+
+/**
+ * Throws error message to end prompting in cli.
+ *
+ * @param errorMessage - error message
+ */
+export function bail(errorMessage: string): void {
+    throw new Error(errorMessage);
+}
+
+/**
+ * Handles error message.
+ *
+ * @param errorMsg - error message
+ */
+export function handleErrorMessage(errorMsg: string): void {
```

Comment:
Should this be named `handleTransportConfigError` as it seems to be specific to that?

Reply:
Thanks, updated